### DISCUSSION TITLE
Small compatibility problem with Rectify11 theme

### DISCUSSION THREAD

**Discussion by emvaized** [ALTaleX531/TranslucentFlyouts#118] [Q&A] [answered]
Hi there! 
I'm using [LightRectified](https://github.com/Rectify11/Installer/tree/master/Resources/Themes/themes) theme from Rectified11 project. And I noticed small incompatibility, which appears only when using both this theme and TranslucentFlyouts (it doesn't happen if either TF is disabled, or system theme switched to default). 

 The problem is: in default File Explorer, when I open any folder and hover an item for the first time, text inside the tooltip goes off-screen on bottom, as if the tooltip bubble is too narrow for the text. When I hover another item, and then the first one, the problem is gone and text fits inside the tooltip correctly. If you open another file, problem occurs again. It also happens with the desktop icons.
 
![illustration](https://github.com/ALTaleX531/TranslucentFlyouts/assets/37851576/955e79eb-f289-4e14-b4aa-5603b48b5436)

I tried to edit the theme file itself using [msstyleEditor](https://github.com/nptr/msstyleEditor) and remove all the padding around the tooltip text, but it didn't help — even without padding, text goes off-screen on bottom when tooltip shows up for the first time. 

But I'm not really sure if this issue is caused by TranslucentFlyouts or by the theme. Maybe you have any idea why it can happen? Maybe the font size of tooltip text changes depending on if it was shown for the first or the second time?

**Comment by ALTaleX531** [comment]
[remove_tooltip_margins.zip](https://github.com/ALTaleX531/TranslucentFlyouts/files/14609863/remove_tooltip_margins.zip)

**Reply by emvaized** [reply]
That was quick! Thank you. 
However, the problem remains after applying these registry keys (along with my modified version of theme with no tooltip padding). Tooltip now has 0 spacing around the text, but the last line still goes off on bottom. 

Any other ideas why it can happen?

**Reply by ALTaleX531** [reply, marked as answer]
I see what's going on, this will be fixed in the next release.

**Reply by emvaized** [reply]
Great! Many thanks.

**Reply by emvaized** [reply]
This problem seems to be fixed in new release v3.1.0. Thanks again!
